### ISSUE TITLE
Wrong template for mux feature in files/build_templates/init_cfg.json.j2

### ISSUE THREAD

**Issue by praveen-li** [Azure/sonic-buildimage#10700]
#### Description

today files/build_templates/init_cfg.json.j2 generates wrong config for mux feature as below:

{
"mux":

{ "auto_restart": "enabled", 
  "has_global_scope": "True", 
  "has_per_asic_scope": "False", 
  "has_timer": "False", 
  "high_mem_alert": "disabled",
  "state": "
{% if 'subtype' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['subtype'] == 'DualToR' %}

enabled{% else %}always_disabled{% endif %}"
  
}
}

#### Steps to reproduce the issue:
1. Perform config reload with any config without feature table.
2.
3.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by samaity** [comment]
@yxieca yes, we got the same output; however, it is taking around 120 sec to get the rendered data in config db. Is it expected? Could you kindly point to any documentation which explains the reasoning behind this logic?
